### DISCUSSION TITLE
Is there anyway to align multi-line math equations with remark-math?

### DISCUSSION THREAD

**Discussion by warrenalphonso** [remarkjs/remark#574] [Q&A] [answered]
`\begin{align}` doesn't work for me, and based on [this comment](https://github.com/remarkjs/remark-math/issues/39#issuecomment-689087797), I don't think it is implemented. Given that, is there any way to align long multi-line equations?

**Comment by wooorm** [comment, marked as answer]
Hi Warren!

Could you provide more info on what you mean doesn’t work? `remark-math` uses dollars around math: `$asd$`. Did you try that?

And what do you mean by align?

**Reply by warrenalphonso** [reply]
Thanks for the quick reply!

<img width="913" alt="Screen Shot 2020-12-03 at 9 26 15 AM" src="https://user-images.githubusercontent.com/38018861/101065142-a316d780-3549-11eb-83b4-027e2436656c.png">

The above screenshot is from my old blog. Notice the equations are aligned so that the equal signs (and also the vertical dots in the third line) are in the same x position. The code for this in MathJax or LaTeX is: 
```
$$ 
\begin{align} 
a^\dagger_1 &= \frac{X - iY}{2} \otimes I \otimes I \otimes ... \otimes I \\ 
a^\dagger_2 &= Z \otimes \frac{X - iY}{2} \otimes I \otimes ... \otimes I \\
&\vdots \\
a^\dagger_{2n} &= Z \otimes Z \otimes Z \otimes ... \otimes \frac{X -iY}{2} 
\end{align}
$$
```

I can't pull this off in remark-math. Anything with `\begin{align}` and `\end{align}` fails to compile.

**Reply by wooorm** [reply]
`remark-math` parses math in markdown, it only cares about the dollars. 💸

What are you using to turn math into something else? rehype-mathjax? rehype-katex? If so, those essentially are thin wrappers around the related projects, so this might be more of a problem for mathjax or katex.

**Reply by warrenalphonso** [reply]
Ah okay. It looks like `align` isn't supported by Katex which was what I was using. Thanks for clearing that up!

For anyone else with this issue, [here](https://katex.org/docs/support_table.html) is a table of supported environments in Katex. Just swap `align` with `aligned` and you get the desired behavior.
